user-guide/Troubleshooting/Procedures/Troubleshooting_SLScripting_Issues.md

```diff
@@ -0,0 +1,7 @@
+---
+uid: TroubleshootingSLScriptingIssues
+---
+
+# Troubleshooting - SLScripting
```

Comment:
#3587 does not link to a separate page, but puts the entire content in this page. Is this going to be aligned?

Reply:
yes, I'll adapt it  in the other PR